requirements.txt

```diff
@@ -26,10 +26,11 @@ PyYAML==5.3.1
 requests==2.23.0
 requests-oauthlib==1.3.0
 rsa==4.0
-sendgrid==6.3.1
+sendgrid==6.4.3
 six==1.14.0
 soupsieve==2.0
 SQLAlchemy==1.3.16
+starkbank-ecdsa==1.0.0
```

Comment:
Did you add this? If so, why?

Reply:
nope didn't add this. if I did I didn't mean to. Ill remove it